config/config.go

```diff
@@ -203,6 +204,29 @@ func (cfg *IncludeConfig) GetExpose() bool {
 	return *cfg.Expose
 }
 
+func (cfg *IncludeConfig) GetMergeStrategy() (MergeStrategyType, error) {
```

Comment:
Is this necessary because you can't set the type of `MergeStrategy` to `*MergeStrategyType` in `IncludeConfig`? I guess the HCL parser doesn't know how to parse a string into that type?

Reply:
Yup... unfortunately.